Cargo.toml

```diff
@@ -249,10 +246,10 @@ unexpected_cfgs = { level = "warn", check-cfg = ['cfg(msim)', 'cfg(fail_points)'
 
 # Dependencies that should be kept in sync through the whole workspace
 [workspace.dependencies]
-mpc = { git = "https://github.com/dwallet-labs/cryptography-private", subdir = "mpc", features = ["parallel"], branch = "pera-network-dkg" }
-class_groups = { git = "https://github.com/dwallet-labs/cryptography-private", branch = "pera-network-dkg", subdir = "class-groups", features = ["default", "threshold", "parallel"] }
-twopc_mpc = { git = "https://github.com/dwallet-labs/cryptography-private", subdir = "2pc-mpc", features = ["secp256k1", "bulletproofs", "paillier", "class_groups", "benchmarking"], branch = "pera-network-dkg" }
-group = { git = "https://github.com/dwallet-labs/cryptography-private", subdir = "group", branch = "pera-network-dkg" }
+mpc = { git = "https://github.com/dwallet-labs/cryptography-private", subdir = "mpc", features = ["parallel"], rev = "36142d48b474211b60c46d380ce41c30981839c9" }
+class_groups = { git = "https://github.com/dwallet-labs/cryptography-private", rev = "36142d48b474211b60c46d380ce41c30981839c9", subdir = "class-groups", features = ["default", "threshold", "parallel"] }
+twopc_mpc = { git = "https://github.com/dwallet-labs/cryptography-private", subdir = "2pc-mpc", features = ["secp256k1", "bulletproofs", "paillier", "class_groups", "benchmarking"], rev = "36142d48b474211b60c46d380ce41c30981839c9" }
+group = { git = "https://github.com/dwallet-labs/cryptography-private", subdir = "group", rev = "36142d48b474211b60c46d380ce41c30981839c9" }
```

Comment:
I suggest we'll keep working with fixed revisions until Scaly's API gets stable, otherwise we'll keep getting errors from api changes

Reply:
Agree. Many things are broken from revision to revision